setup.py

```diff
@@ -9,15 +9,15 @@
     readme = readme_file.read()
 
 requirements = [
-    "aicspylibczi>=2.5.0",
```

Comment:
I don't think there are supposed to be spaces in this string... I am surprised this worked.

Reply:
I just removed the spaces. assuming the tests pass I'll merge it into master.